widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadPositionHolder.java

```diff
@@ -0,0 +1,21 @@
+package com.glia.widgets.view.head;
+
+import androidx.core.util.Pair;
+
+public class ChatHeadPositionHolder {
```

Comment:
why not just pojo like `class ChatHeadPosition` with setX/getX methods? Currently `ChatHeadPositionHolder` makes me think theres something more advanced going on than just plain old java object.